README.md

```diff
@@ -1,8 +0,0 @@
-![Bits of Good](/assets/header.png)
```

Comment:
Please undo this file deletion

Reply:
Is this the README for our cohort or the BoG README?